TYPO3.Neos/Documentation/CreatingASite/ContentCache.rst

```diff
@@ -182,6 +183,19 @@ The ``ContentCollection`` cache configuration declares a tag that will flush the
 any of it's descendants (direct or indirect child) changes. So editing a node inside the collection will flush the
 whole collection cache entry and cause it to re-render.
 
+.. note::
+  When using ``cached`` as the cache mode, your ``entryTags`` should always contain the node identifier. Otherwise, the
+  cache will not be flushed when you make changes to the node itself, which will lead to unexpected behavior in the Neos
```

Comment:
Also, would there be any disadvantages (such as worse caching performance) when you include the node identifier in such cases? I think there wouldn't, no? In that case I'd say let's stick with the original proposal, otherwise I'm happy to include a sentence on that case @dimaip ;)